package.json

```diff
@@ -69,6 +69,7 @@
     "@types/node": "^18.0.0",
     "@typescript-eslint/eslint-plugin": "^4.27.0",
     "@typescript-eslint/parser": "^4.27.0",
+    "angular-mocks": "*",
```

Comment:
**question:** What is the need for this package?

Reply:
it was to run unit testing with mocks. i removed this